test/logflare_web/controllers/api/account_controller_test.exs

```diff
@@ -0,0 +1,42 @@
+defmodule LogflareWeb.Api.AccountControllerTest do
+  use LogflareWeb.ConnCase
+
+  setup do
+    [token: token] =
+      Application.get_env(:logflare, LogflareWeb.Plugs.EnsureSuperUserAuthentication)
+
+    {:ok, %{token: token}}
+  end
+
+  describe "create/2" do
+    test "returns 201 and the user information and a token to access the API", %{
+      conn: conn,
+      token: token
+    } do
+      email = Faker.Internet.email()
```

Comment:
see above comment, for removing faker usage

Reply:
🤦 nice catch x2